main/signers/Signer/index.ts

```diff
@@ -77,7 +77,7 @@ export default class Signer extends EventEmitter {
       name: this.name || this.type + ' signer',
       type: this.type,
       model: this.model,
-      addresses: this.addresses,
+      addresses: this.addresses.map(addr => addr.toString()),
```

Comment:
Frame displays addresses returned with a `0x` prefix, meaning it's being adding elsewhere. Seems that's sufficient and we don't need to add it ourselves.

Reply:
The `0x` prefix is added whenever you make a request to get a formatted address (which we have always done with Frame).

I brought it up because in a recent version we overloaded this method to allow request of pubkeys, which are returned as buffers.

We're going to split that out in a future SDK version. Just wanted to make note of the distinction here and point to the `addr.toString()` casting being unnecessary.

Reply:
Added `0x` prefix with this [commit](https://github.com/floating/frame/pull/845/commits/34be0f1d46b648d4b2c62449651cfd6cc7aa1eb4). Thank you, @douglance 🥇